infra/stack/rds.ts

```diff
@@ -0,0 +1,25 @@
+import * as pulumi from "@pulumi/pulumi";
+import * as aws from "@pulumi/aws";
+
+export function createDBInstance() {
+  const dbSecret = new aws.kms.Key("example", { description: "Example KMS Key" });
+
+  const db = new aws.rds.Cluster("postgresql", {
+    availabilityZones: [
+      "eu-west-1a",
+      "eu-west-1b",
+      "eu-west-1c",
+    ],
+    backupRetentionPeriod: 5,
```

Comment:
@kudaMhizha going with Aurora here ?

Reply:
@kudzai-sovtech nah was just testing it out, what do you think?

Reply:
@kudaMhizha just thinking about the cost, but we're just testing.